### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by jycr753**
This version of jose passes all unit testing

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
         "illuminate/contracts": "5.1.* || 5.2.*",
         "illuminate/http": "5.1.* || 5.2.*",
         "illuminate/support": "5.1.* || 5.2.*",
-        "namshi/jose": "^6.0",
+        "namshi/jose": "^7.1",
```
Comment (lucasmichot) [inline composer.json]: `"^7.0"` is enough

Comment (jycr753) [inline composer.json]: enough? is there a good reason why not update to the latest?

Comment (lucasmichot) [inline composer.json]: People might already use `7.0.0` in their project - forcing to use `^7.1` would cause package conflicts for them.

**Comment by lucasmichot** [comment]
You could probably just reset to 2eb7c442e2b6a9e51f9aa6eabb67056e7b0094d1 and `push --force` it, as https://github.com/tymondesigns/jwt-auth/pull/679/commits/b402b71607f370ad8c59afa37011a5effe5427fb just reverses https://github.com/tymondesigns/jwt-auth/pull/679/commits/2eb7c442e2b6a9e51f9aa6eabb67056e7b0094d1

**Comment by jycr753** [comment]
Yeah sorry just created another one with the right things, sorry about that

**Pull request closed without merging**